resources/api/openapi.yml

```diff
@@ -1,454 +1,696 @@
-                                # https://github.com/OAI/OpenAPI-Specification/issues/1706
-                                style: deepObject
-                                explode: true
-            responses:
-                200:
-                    description: 'Expected response to a valid request.'
-                    content:
-                        application/json:
-                            schema:
-                                $ref: '#/components/schemas/InitializePaymentResponse'
-                400:
-                    description: 'Bad request.'
-                default:
-                    description: 'Expected response to a bad request.'
-                    content:
-                        text/plain:
-                            schema:
-                                type: string
+  '/private/pre-order-payment':
+    post:
+      operationId: 'preOrderPayment'
+      x-spryker:
+        module: PaymentBackendApi
+        controller: PreOrderPaymentResourceController
+      summary: 'Initializes a payment with a third party provider before an order is persisted.'
+      security:
+        - Bearer: [ ]
+      parameters:
+        - $ref: '#/components/parameters/tenantIdentifier'
+      requestBody:
+        description: 'Provides Order Data (incl. orderReference) or Quote Data (excl. orderReference).'
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PreOrderPaymentRequest'
+          # SCOS tenants are NOT sending this as JSON, but as form data! This is only for BC reasons. "application/json" should be used instead.
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/PreOrderPaymentResponse'
+            encoding:
+              orderData:
+                # Arrays of objects (as url-encoded form data) is not officially supported by the
+                # OpenAPI spec, as there are different standards for how this is serialized.
+                # The current format will incorrectly appear as a JSON form field in swagger, while
+                # postman will generate working examples with it.
+                # See this discussion on the OpenAPI specification on the limitations of deepObject
+                # https://github.com/OAI/OpenAPI-Specification/issues/1706
+                style: deepObject
+                explode: true
+      responses:
+        200:
+          description: 'Expected response to a valid request.'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InitializePaymentResponse'
+        400:
+          description: 'Bad request.'
+        default:
+          description: 'Expected response to a bad request.'
+          content:
+            text/plain:
+              schema:
+                type: string
```

Comment:
what is the content type for `400:`? would be good to have an example (or generic schema for errors)

why `default:` has description `'Expected response to a bad request.'`?

![image](https://github.com/user-attachments/assets/18d389f7-9bb6-44f3-874a-24ae6bbdbc37)


Suggestion: replace default with `5XX`

```suggestion
400:
          description: 'Bad request.'
          content:
            application/json:
              schema:
                type: object
        '5XX':
          description: 'Unexpected error.'
```